projects/driver_controls/src/can_output.c

```diff
@@ -0,0 +1,26 @@
+#include "can_output.h"
+#include "event_arbiter.h"
+#include "input_event.h"
+#include "log.h"
+
+// CAN device ids as defined in confluence
```

Comment:
Aren't these all message ids?

Also, I'm assuming that you'll be using the codegen values once that gets integrated?

Reply:
Yep. Once codegen is done, I'll update the module accordingly.